scipy/stats/_hypotests.py

```diff
@@ -1433,3 +1435,242 @@ def cramervonmises_2samp(x, y, method='auto'):
+    >>> for ((i, j), p) in np.ndenumerate(res.pvalue):
+    ...     # filter out self comparisons
+    ...     if i != j:
+    ...         print(f"({i} - {j}) {p:.03f}")
+    (0 - 1) 0.012
+    (0 - 2) 0.980
+    (1 - 0) 0.012
+    (1 - 2) 0.017
+    (2 - 0) 0.980
+    (2 - 1) 0.017
+
+    The null hypothesis is that each group has the same mean. We choose a
+    significance level of .05. The p-value for comparisons between ``group0``
+    and ``group1`` as well as ``group1`` and ``group2`` do not exceed .05, so
+    we reject the null hypothesis that they have the same means. The p-value of
+    the comparison between ``group0`` and ``group2`` exceeds .05, so we accept
+    the null hypothesis that there is not a significant difference between
+    their means.
+
+    There are also confidence intervals associated with the test statistic.
+    They are applicable for the 95% confidence level becuase the test was
+    applied with a .05 significance level. Print out the lower confidence
+    interval for this result:
+
+    >>> conf = res.confidence_interval()
+    >>> for ((i, j), l) in np.ndenumerate(conf.low):
+    ...     # filter out self comparisons
+    ...     if i != j:
+    ...         h = conf.high[i,j]
+    ...         print(f"({i} - {j}) {l:>6.3f} {h:>6.3f}")
+    (0 - 1) -8.249 -0.951
+    (0 - 2) -3.909  3.389
+    (1 - 0)  0.951  8.249
+    (1 - 2)  0.691  7.989
+    (2 - 0) -3.389  3.909
+    (2 - 1) -7.989 -0.691
+    """
+    args = _tukeykramer_iv(args, sig_level)
+    nargs = len(args)
+    means = np.asarray([np.mean(arg) for arg in args])
+    nsamples_args = np.asarray([a.size for a in args])
+    nsamples = np.sum(nsamples_args)
+
+    # determine mean square error
+    mse = (np.sum([np.var(arg, ddof=1) for arg in args] *
+                  (nsamples_args - 1)) / (nsamples - len(args)))
+
+    # The calculation of the standard error differs when treatments differ in
+    # size. See ("Unequal sample sizes")[1].
+    if np.unique(nsamples_args).size == 1:
+        # all input groups are the same length, so only one value needs to be
+        # calculated [1].
+        normalize = 2 / nsamples_args[0]
+    else:
+        # to compare groups of differing sizes, we must compute a variance
+        # value for each individual comparison. Use broadcasting to get the
+        # resulting matrix. [3], verified against [4] (page 308).
+        normalize = 1 / nsamples_args + 1 / nsamples_args.reshape((nargs, 1))
+
+    # the standard error is used in the computation of the tukey criterion and
+    # finding the p-values.
+    stand_err = np.sqrt(normalize * mse / 2)
+
+    # the mean difference is the test statistic.
+    mean_differences = means.reshape((nargs, 1)) - means
+
+    # Calculate the t-statistic to use within the survival function of the
+    # studentized range to get the p-value.
+    t_stat = np.abs(mean_differences) / stand_err
+
+    params = t_stat, nargs, nsamples - 1
```

Comment:
```suggestion
    params = t_stat, nargs, nsamples - nargs
```
No? 
If this change is right, I wonder how this slipped past tests, please devise a test that is sensitive enough that it would have caught this.

Reply:
Unfortunately, both `-1` and `-nargs` seem to pass just fine. I will have to find a test case where `nargs` is very much different than one. 

I suspect it is because the difference between the total number of observations across all groups less 1 or 3 is small. I'll create a test with small treatments to hopefully catch this.



<img width="226" alt="Screen Shot 2021-06-02 at 3 32 08 PM" src="https://user-images.githubusercontent.com/44255917/120560524-b2bb8f80-c3b7-11eb-9a8b-ec36fa434935.png">

Reply:
Actually, increasing the precision of the R test illustrates that this was an issue of why it was only accurate to 3 digits for the p-value. I can now increase it.